spec/helpers/logging_helper_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe LoggingHelper do
+
+  describe '#logging_messages' do
+    context 'logs a message'
+    it {
+      expect(Rails.logger).to receive(:info).with(/event=test_event/)
+      LoggingHelper.log_event(LoggingHelper.current_request, LoggingHelper.current_user, {event: "test_event"})
```

Comment:
Will current_request and current_user be nil here? Perhaps you should test with real values?